pallets/eth-bridge/src/tests/mock.rs

```diff
@@ -181,8 +184,28 @@ pub fn create_confirming_author(author_id: u64) -> Author<TestRuntime> {
 
 pub fn create_mock_event_partition(events: EthEvent) -> EthereumEventsPartition {
     let mut partition: BoundedBTreeSet<DiscoveredEvent, EventsBatchLimit> = BoundedBTreeSet::new();
-    partition.try_insert(DiscoveredEvent { event: events.clone(), block: 2 }).unwrap();
-    EthereumEventsPartition::new(EthBlockRange {start_block: 1, length: 1000}, 0, false, partition)
+    partition
+        .try_insert(DiscoveredEvent { event: events.clone(), block: 2 })
+        .unwrap();
+    EthereumEventsPartition::new(
+        EthBlockRange { start_block: 1, length: 1000 },
+        0,
+        false,
+        partition,
+    )
+}
+
+pub fn create_second_mock_event_partition(events: EthEvent) -> EthereumEventsPartition {
```

Comment:
You can update `create_mock_event_partition` to take an extra argument. This avoids creating a second function.